### ISSUE TITLE
[v6] Calling `getAll` in Node 15+ fails

### ISSUE THREAD

**Issue by asyarb** [prismicio/prismic-javascript#176]
Calling `client.getAll()` and other `getAll` variants like `getAllByType` inside of `getStaticProps` cause the development server to hang indefinitely.

**Example Code**

```tsx
export async function getStaticProps() {
  let client = createClient(getEndpoint(""), { accessToken: "", })

  // All of these variations never resolve:
  let allFaqs = await client.getAllByType("faq")
  // let allFaqs = await client.getByType("faq")
  // let allFaqs = await client.getAll({
  //   predicates: predicate.at("document.type", "faq"),
  // })

  return {
    props: {
      allFaqs,
    },
  }
}
```

**Package Versions**
- @prismicio/client `6.0.0-alpha.1`
- Next `10.2.3`
- React `17.0.2`

Let me know if you need any additional information or details!

**Comment by asyarb** [comment]
On further discussion and investigation with @angeloashmore (thanks!), it appears that this issue is specific to Node 15+. Outside of NextJS, the following also fails in pretty straight-forward script:

```js
import fetch from "node-fetch"
import { createClient, getEndpoint } from "@prismicio/client"

async function main() {
  let client = createClient(getEndpoint(""), { accessToken: "", fetch })

  let faqs = await client.getAllByType("faq")
  console.log(faqs)
}

main()
```

**Comment by angeloashmore** [comment]
This is fixed in `v6.0.0-beta.2`. The library should now work in Node 12+.
